pkg/apis/mxnet/v1/defaults.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	commonv1 "github.com/kubeflow/common/pkg/apis/common/v1"
```

Comment:
Please move this import stmt to the next group.

Reply:
em. I didn't enable `goimport` file watchers in my IDE. Let me update it